lib/riemann/tools/apache_status.rb

```diff
@@ -14,9 +14,10 @@ class ApacheStatus
       require 'uri'
 
       opt :uri, 'Apache Server Status URI', default: 'http://localhost/server-status'
+      opt :user_agent, 'User-Agent header to use for HTTP requests', short: :none, default: "#{File.basename($PROGRAM_NAME)}/#{Riemann::Tools::VERSION} (+https://github.com/riemann/riemann-tools)"
```

Comment:
Minor nit. I'd chop the "to use" out of this - shorten it.

Reply:
Same for all other uses, etc.

Reply:
Did that and added the parameter that slept through for consul.